plugin/settings.gradle.kts

```diff
@@ -0,0 +1,2 @@
+rootProject.name = "Splitties-Plugin"
```

Comment:
Can't it be made a sub-project of the existing root project? Or is there reasons to not do it?
I'd like to share the library version (located in root project's `gradle.properties`) with the `Splitties` object's constants.